Modix.Data/Repositories/BehaviourConfigurationRepository.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using Modix.Data.Models;
+
+namespace Modix.Data.Repositories
+{
+    public interface IBehaviourConfigurationRepository
+    {
+        Task<IEnumerable<BehaviourConfiguration>> GetBehaviours();
```

Comment:
Should be `GetAsync` for consistency.

Reply:
There's no synchronous implementation - suffixing `Async` onto all methods that return `Task` or `Task<T>` is odd? The compiler would let you know enough?

Reply:
Again, I don't really care either way, but we should be consistent throughout the codebase.